### PR TITLE
chore(e2e): Add sensor scanner support to gke flavor of CI for ui-e2e

### PR DESCRIPTION
**Pull request by vjwilson**
## Description

What: Add sensor scanner support to gke flavor of CI for ui-e2e

Why: To allow the new tests for new delegated registries scanning UI to run the same on GKE envs, as well as OCP envs.

## Checklist
- [x] Investigated and inspected CI test results (failure of unrelated ocp-qa-e2e-test is flake)

## Testing Performed

CI

### CODE REVIEW

**Review by connorgorman — APPROVED** [review]

**Review by dvail — APPROVED** [review]

File: scripts/ci/jobs/gke_ui_e2e_tests.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 # Override test env defaults here:
 # (for defaults see: tests/e2e/lib.sh export_test_environment())
 os.environ["OUTPUT_FORMAT"] = "helm"
+os.environ["SENSOR_SCANNER_SUPPORT"] = "true"
```
Comment (RTann) [inline scripts/ci/jobs/gke_ui_e2e_tests.py:20]: is this new? I never heard of it before

Comment (vjwilson) [reply scripts/ci/jobs/gke_ui_e2e_tests.py:20]: @dcaravel turned me on to it. It triggers the install of a Secured Cluster scanner, to be used for delegating scanning duties to the cluster where the image is used, or for scans invoked manually through roxctl, when the customer has locked down their private registry/registries to only be accessible from certain clusters.

**Review by janisz — APPROVED** [review]

**Review by davdhacs — APPROVED** [review]
+1

**Comment by vjwilson** [comment]
I got blessing from the ACS Automation team (@BradLugo and @davdhacs ) to merge, despite failing ocp-qa-e2e-test.

**Pull request merged**